scripts/windowsScript.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Why do we need both scripts?

Reply:
Peter asked me to include both scripts for now as he didn't build our plugin on jenkins yet. On Friday he built it and confirmed that we can remove .sh script. I removed it now.